2-mandatory/2-retrying-random-numbers.js

```diff
@@ -11,6 +11,11 @@ function generateRandomNumber() {
 
 function getRandomNumberGreaterThan50() {
     // TODO - implement using a do-while loop
+    let num;
+    do {
+        num = generateRandomNumber();
+    } while (num <= 50);
```

Comment:
A small thing, the exercise says only smaller than 50 not smaller than or equal so yeah

Reply:
Thank you for your feedback Altom. I am fixing it.